.github/workflows/cargo-generate.yaml

```diff
@@ -0,0 +1,110 @@
+# This workflow is used to automatically update the `cargo-generate` branch. If you copied the
+# template, you can safely delete this. :)
+#
+# BE VERY CAREFUL about what this workflow runs, since IT HAS THE ABILITY TO DELETE THE ENTIRE
+# REPOSITORY'S CONTENTS. For reason, stick to pre-installed tools and trusted actions (such as
+# those hosted from the `actions` organization.)
+
+name: Update `cargo-generate` branch
+
+on:
+  # Update `cargo-generate` from `main` every time `main` is updated.
+  push:
+    branches: main
+  # Allow this workflow to be manually triggered.
+  workflow_dispatch:
+
+# This workflows also has write-access for content and workflow, but it is through
+# CARGO_GENERATE_PAT.
+permissions:
+  # We need write permissions to create new issues.
+  issues: write
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    # Only run this job on the source repository, skipping forks.
+    if: ${{ github.repository == 'TheBevyFlock/bevy_quickstart' }}
```

Comment:
Really good idea!

Reply:
Yeah! I'm not sure if it's possible to exclude this file from users of this template, so this is the safeguard.

Reply:
Maybe `cargo-generate.yaml` can be put in the `cargo-generate` branch? I'm not sure if it would still trigger properly on commits to `main` though.

Reply:
It wouldn’t, unfortunately, though I like where your head is at!

In the end, it’s probably not too big of a deal. I put comments saying users can safely delete it, and it won’t run anything anyways on other repos.